contexts/arbo-context.tsx

```diff
@@ -11,7 +11,7 @@ import {
   getBoundingBoxFromCountryName,
 } from "@/lib/country-bounding-boxes";
 import useArboData from "@/hooks/useArboData";
-import { parseISO } from "date-fns";
+import { parseISO, isAfter, isBefore } from "date-fns";
```

Comment:
:heart: :heart: :heart:
I love using these functions over `<` and `>` as well but it looks like you might have imported them and forgot to use them?

Reply:
Oh yeah I tried to use those and for some reason it was giving me invalid dates I think it might be a specific formatting issue. I read the docs and was not entirely sure what they wanted.